ReactSkia/utils/RnsJsRaf.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include<iostream>
+
+#include <jsi/JSIDynamic.h>
+#include <jsi/instrumentation.h>
+
+#include "ReactSkia/utils/RnsLog.h"
+
+using namespace facebook::jsi;
+
+namespace facebook {
+namespace react {
+
+class RnsJsRequestAnimation {
+  public:
+    RnsJsRequestAnimation(const std::function<void(double)>& callback);
+    virtual ~RnsJsRequestAnimation() { };
+
+    void start();
+    void stop();
+
+    static uint64_t nextUniqueId();
+
+  private :
+    bool isActive_{false};
```

Comment:
can we make isActive_ type as atomic bool , since it will be accessed by two different threads 
std::atomic<bool>

Reply:
Addressed